lib/generators/react_on_rails/install_generator.rb

```diff
@@ -44,13 +46,18 @@ class InstallGenerator < Rails::Generators::Base
                    aliases: "-b"
 
       def run_generators # rubocop:disable Metrics/CyclomaticComplexity
-        return unless installation_prerequisites_met?
+        return output_errors unless installation_prerequisites_met?
```

Comment:
I really don't like this as output_errors looks like a return value.

1. Let's rename the method to `puts_errors`

```
unless installation_prerequisites_met?
  print_errors
  return
end
```

@samnang @mapreal19 @alexfedoseev 

How do feel about the pattern of `return fn` where `fn` returns nil.